modules/ibc-hooks/README.md

```diff
@@ -0,0 +1,351 @@
+# IBC-hooks
+
+> This module is forked from https://github.com/osmosis-labs/osmosis/tree/main/x/ibc-hooks
+
+## IBC hooks
```

Comment:
I used the name "Wasm Hooks" in the docs before because the ibc hooks could theoretically accept hooks other than the provided WasmHook. That said, it probably doesn't matter unless someone else adds other hook types and adds them to the module. We can update the docs then